esp-hal/src/gpio/mod.rs

```diff
@@ -2475,6 +2475,7 @@ mod asynch {
 
     use super::*;
 
+    #[ram]
```

Comment:
Was this intentional? (Wondering if it was a temporary change for a local test)

Reply:
This one was intentionally - while the waker's internals are places in RAM the array was placed in flash otherwise